src/pg/types.ts

```diff
@@ -86,6 +109,17 @@ export type DbJob = {
   updated_at?: string;
 };
 
+export type DbUpdateNotification = {
+  token_id: number;
```

Comment:
Token-id is type `number` here, just a heads up in case it needs to handle larger int sizes